src/api.ts

```diff
@@ -103,34 +183,42 @@ export async function fetchApi <
 	{
 		if (!response.ok)
 		{
-			logger.error("Got success response, but API response is no success");
+			logger?.error("Got success response, but API response is no success");
 		}
 
 		// @ts-expect-error .data errors out in zod right now. So we add this and the cast in the meantime
 		return successResponse.data.data as z.infer<typeof dataSchema>;
 	}
-	else
-	{
-		logger.debug("No success cause", successResponse.error);
-	}
 
+	console.log(responseData);
```

Comment:
Is this `console.log` intentional or can it be removed?

Reply:
Nope, removed